examples/00-EDB/14_edb_create_parametrized_design.py

```diff
@@ -47,23 +47,59 @@
 simulation_configuration.step_freq = "10MHz"
 
 ##########################
-# Build simulation project
-# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Now apply the simulation setup to the EDB.
 
 edb.build_simulation_project(simulation_configuration)
 
 #############################
-# Generated design parameters
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ### Parameterization
+#
+# The layout can automatically be set up to enable parametric studies. For example, the
+# impact of antipad diameter or trace width on signal integrity performance may be invested parametrically.
```

Comment:
```suggestion
# The layout can automatically be set up to enable parametric studies. For example, the
# impact of the anti-pad diameter or trace width on signal integrity performance may be parametrized.
```
Earlier, antipad was hyphenated.  We want to be consistent. It it should be one word, which the name of the paramter is one word, please change the earlier occurrences.